packages/config/src/ios/Updates.ts

```diff
@@ -37,27 +52,29 @@ export function setUpdatesConfig(
 ) {
   let newExpoPlist = {
     ...expoPlist,
-    EXUpdatesEnabled: getUpdatesEnabled(config),
-    EXUpdatesURL: getUpdateUrl(config, username),
-    EXUpdatesCheckOnLaunch: getUpdatesCheckOnLaunch(config),
-    EXUpdatesLaunchWaitMs: getUpdatesTimeout(config),
+    [Config.ENABLED]: getUpdatesEnabled(config),
+    [Config.UPDATE_URL]: getUpdateUrl(config, username),
+    [Config.CHECK_ON_LAUNCH]: getUpdatesCheckOnLaunch(config),
+    [Config.LAUNCH_WAIT_MS]: getUpdatesTimeout(config),
   };
 
-  const updateUrl = getUpdateUrl(config, username);
-  if (updateUrl) {
-    newExpoPlist = {
-      ...newExpoPlist,
-      EXUpdatesURL: updateUrl,
-    };
-  }
-
-  const sdkVersion = getSDKVersion(config);
-  if (sdkVersion) {
+  const runtimeVersion = getRuntimeVersion(config);
+  if (runtimeVersion) {
```

Comment:
```suggestion
  const runtimeVersion = getRuntimeVersion(config);
  const sdkVersion = getSDKVersion(config);
  if (runtimeVersion) {
    ...
  } else if (sdkVersion) {
    ...
  } else {
    throw error 
  }
```
I'm not sure what the proper behaviour in the else statement should eb but we should handle it